package.json

```diff
@@ -27,7 +27,7 @@
     "@typescript-eslint/eslint-plugin": "^6.5.0",
     "@typescript-eslint/parser": "^6.5.0",
     "eslint": "^8.48.0",
-    "eslint-config-agile-digital": "^3.3.0",
```

Comment:
If we're renaming, we should publish a final version with this package name that deprecates it https://docs.npmjs.com/deprecating-and-undeprecating-packages-or-package-versions#deprecating-an-entire-package